README.md

```diff
@@ -1,15 +1,39 @@
 # leah-feelix-project
 
+## To start the app
+
+1. clone the repo to your local machine
+2. change directory to "contacts-app"
+3. run app locally:
+
+```
```

Comment:
maybe say you need to also run npm install

Reply:
done